test-poc/framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -1,15 +1,50 @@
 package org.keycloak.test.framework.server;
 
+import org.keycloak.it.utils.RawKeycloakDistribution;
+
+import java.util.LinkedList;
+import java.util.List;
+
 public class DistributionKeycloakTestServer implements KeycloakTestServer {
 
+    private boolean debug = false;
+    private boolean manualStop = true;
+    private boolean enableTls = false;
+    private boolean reCreate = false;
+    private boolean removeBuildOptionsAfterBuild = false;
+    private int requestPort = 8080;
+    private RawKeycloakDistribution keycloak;
+
     @Override
     public void start(KeycloakTestServerConfig serverConfig) {
-        throw new RuntimeException("Method not implemented!");
+        keycloak = new RawKeycloakDistribution(debug, manualStop, enableTls, reCreate, removeBuildOptionsAfterBuild, requestPort);
+
+        // Set environment variables user and password for Keycloak Admin used by Keycloak instance.
+        keycloak.setEnvVar("KEYCLOAK_ADMIN", serverConfig.adminUserName().get());
+        keycloak.setEnvVar("KEYCLOAK_ADMIN_PASSWORD", serverConfig.adminUserPassword().get());
```

Comment:
This style of environment variables will be deprecated, see #31114. Perhaps we can use the newer variants?